src/templates/bootstrap/formio-component.html

```diff
@@ -3,6 +3,6 @@
         <button *ngIf="component.multiple && (i > 0)" class="btn btn-danger" style="position:absolute;right:0;" (click)="removeAt(i)"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></button>
         <formio-element [form]="form" [component]="comp" [label]="label" [events]="events" [render]="render"></formio-element>
     </div>
-    <button *ngIf="component.multiple" type="button" class="btn btn-default" (click)="addComponent()"><span class="glyphicon glyphicon-plus" aria-hidden="true"></span></button>
+    <button *ngIf="component.multiple && component.type!='select'" type="button" class="btn btn-default" (click)="addComponent()"><span class="glyphicon glyphicon-plus" aria-hidden="true"></span></button>
```

Comment:
I would rather have a method defined in the component class that determines if this button should be visible. Something like 

'''*ngIf="component.allowMultiple ()"'''

Then the base class would return '''return this.multiple;''' while the select component would override the base and return false.